docs/memory/sgx-config.md

```diff
@@ -0,0 +1,20 @@
+# SGX Resource Configurator Plugin
+
+This sample allows configuring SGX resources using annotations.
+
+## SGX Annotations
+
+You can annotate SGX EPC configuration for every container in the pod,
+or just a specified container using the following annotation notations:
+```
+...
+metadata:
+  annotations:
+    # for all containers in the pod
+    sgx-epc.nri.io/pod: "32768"
+    # alternative notation for all containers in the pod
+    sgx-epc.nri.io: "8192"
+    # for container c0 in the pod
+    sgx-epc.nri.io/container.c0: "16384"
```

Comment:
As @askervin already mentioned in a [comment](https://github.com/containers/nri-plugins/pull/156#pullrequestreview-1681743401) the scope of this plugin will be maybe extended. Therefore the current annotation key `sgx-epc.nri.io` could be misleading in the future. A more specific annotation key could be beneficial. Moreover, maybe the annotation name could be aligned to the current key name `sgx.intel.com/epc` defined in the [Intel SGX device plugin repository](https://github.com/intel/intel-device-plugins-for-kubernetes/tree/main#intel-sgx-admission-webhook) and in intel/intel-device-plugins-for-kubernetes#1567, e.g.
```suggestion
    # for all containers in the pod
    sgx.intel.com/epc-limit-pod: "32768"
    # alternative notation for all containers in the pod
    sgx.intel.com/epc-limit: "8192"
    # for container c0 in the pod
    sgx.intel.com/epc-limit-container.c0: "16384"
```

Reply:
Since this is a community maintained repository I'd like to avoid any vendor names or other such touchy or iffy subjects in all notations.

That said I also disliked the sgx-epc annotation as sounding mostly gibberish for the uninitiated (myself included), so I tried to come up with a better annotation name. I ended up with `epc-limit` for now (since I found my preferred `encrypted-page-cache-limit` a bit too long). Better names are still welcome.

Reply:
@klihub I fully agree that vendor specific names should not be used here. I'm sorry that I overlooked that.

However, I think plugin name should be reflected as a sub-namespace in the annotation key. The other plugins do the same. Maybe

```yaml
    # for all containers in the pod
    limit.sgx-epc.nri.io/pod: "32768"
    # alternative notation for all containers in the pod
    limit.sgx-epc.nri.io: "8192"
    # for container c0 in the pod
    limit.sgx-epc.nri.io/container.c0: "16384"
```

or

```yaml
    # for all containers in the pod
    epc-limit.sgx.nri.io/pod: "32768"
    # alternative notation for all containers in the pod
    epc-limit.sgx.nri.io: "8192"
    # for container c0 in the pod
    epc-limit.sgx.nri.io/container.c0: "16384"
```

What do you think?